contracts/transmuter/src/contract.rs

```diff
@@ -409,6 +674,61 @@ impl Transmuter<'_> {
 
         Ok((pool, token_in))
     }
+
+    // --- admin ---
+
+    #[msg(exec)]
+    pub fn transfer_admin(
```

Comment:
Not transmuter specific, but there are a few patterns that we keep using. It would be good to make sylvia interfaces that we can reuse on all osmosis contracts

Reply:
Good idea, I think this is simlilar to openzeppelin [Ownable](https://docs.openzeppelin.com/contracts/3.x/access-control), we might wanna have our own collection.

Reply:
100%! we could even add some osmosis-specific stuff behind compilation flags (so they're only included when needed)